ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -1,5 +1,31 @@
 presubmits:
   openshift/release:
+  - agent: kubernetes
+    always_run: false
```

Comment:
Huh why? Does this take this much time?

Reply:
I thought `always_run` must be false if `run_if_changed` is set.

So `always_run` means for every PR it runs.
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md

Updated.

Reply:
Ha. I was right.

```yaml
{"component":"unset","error":"error loading prow config: job pull-ci-openshift-release-ci-secret-bootstrap-config-validation is set to always run but also declares run_if_changed targets, which are mutually exclusive","file":"prow/cmd/checkconfig/main.go:210","func":"main.main","level":"fatal","msg":"Validation failed","severity":"fatal","time":"2021-02-25T13:22:26Z"}
```

Reply:
Removed `run_if_changed` as well.
I just feel we do not need to run it if the config is not changed.

Reply:
Heh, I wouldn't bother if the thing runs fast /shrug